### PR TITLE
Fix configuration paths

### PR DESCRIPTION
**Pull request by madhawa-gunasekara**
This fix is added to get the configuration folder, using kernel configurations. without hardcoding the path

### CODE REVIEW

File: components/event/org.wso2.carbon.event.core/src/main/java/org/wso2/carbon/event/core/internal/builder/EventBrokerBuilder.java [hunk 1]
```diff
@@ -74,8 +74,8 @@ public static EventBroker createEventBroker() throws EventBrokerConfigurationExc
      */
     private static OMElement loadConfigXML() throws EventBrokerConfigurationException {
 
-        String carbonHome = System.getProperty(ServerConstants.CARBON_HOME);
-        String path = carbonHome + File.separator + "repository" + File.separator + "conf" + File.separator + EventBrokerConstants.EB_CONF;
+        String carbonConfigHome = System.getProperty(ServerConstants.CARBON_CONFIG_DIR_PATH);
```
Comment (Niranjan-K) [inline components/event/org.wso2.carbon.event.core/src/main/java/org/wso2/carbon/event/core/internal/builder/EventBrokerBuilder.java]: In the previous implementation, the path was constructed. If the System property is not defined, then there is an issue.

Comment (madhawa-gunasekara) [inline components/event/org.wso2.carbon.event.core/src/main/java/org/wso2/carbon/event/core/internal/builder/EventBrokerBuilder.java]: but Normally we set this property in server startup file.

**Comment by Niranjan-K** [comment]
The PR subject be imperative, i.e., it should be "Fix" and not "fixing"

**Comment by madhawa-gunasekara** [comment]
Thanks @Niranjan-K  I have done the changes to the commit as well

**Comment by madhawa-gunasekara** [comment]
@Niranjan-K @Asitha @anupama-pathirage 

Thanks

**Pull request merged**